Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3-alpine
+
+ARG PHOEBE_BRANCH=master
+
+RUN apk add --no-cache \
+      git \
+      build-base \
+      musl-dev
+
+RUN git clone --branch ${PHOEBE_BRANCH} --single-branch --depth 1 \
+      https://github.com/phoebe-project/phoebe2.git /phoebe
```

Comment:
doesn't running this file require to already have cloned the repo though?  Would it make more sense to either keep this external or ship it with phoebe but have the branch handling done via git outside of this?

Reply:
This clones the latest version of the repo branch *inside* the docker image. We could copy the local repo but then you would need to switch branches manually and any local changes might trickle in. I think this is cleaner.